spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -92,6 +93,7 @@ The fungible token transfer bridge module tracks escrow addresses associated wit
 ```typescript
 interface ModuleState {
   channelEscrowAddresses: Map<Identifier, string>
+  channelForwardingAddresses: Map<Identifier, string>
```

Comment:
I agree, we could use directly channelEscrowAddresses.  The channelEscrowAddresses mapping is populated during `onChainOpenTry` at line 181. Instead the channelForwardingAddresses is never set. Thus the read we do at line 331 may be illegal.

Reply:
note https://github.com/cosmos/cosmos-sdk/issues/7737#issuecomment-735671951

Reply:
The forwarding address is mainly just used as pass through storage based on how ics20 is constructed in the non-forwarding case. I think the spec just needs to specify that this address should be an address the transfer module controls. I don't think it should reuse the escrow addresses

Reply:
Unless I am missing something, I can't think of a reason why you need channel isolated forwarding addresses https://github.com/cosmos/ibc-go/issues/6561#issuecomment-2176403684

Reply:
The reason I decided to split it off to a separate address is just an additional security blanket. The balance of the escrow addresses is security critical. If the balance ends up in an invalid state, that can be a very painful issue to unwind.

It should be completely fine if there's no bugs. But if there's a bug in forwarding, i figure having a separate account might make the impact a bit more isolated.

Of course it is receiving from and sending back to the escrow addresses, so if there's issues there it could still be a problem for escrow account balances.

I'm ok either way. I just figured to start out with a separate account for clarity and avoiding having the escrow account representing more than one thing semantically.

We can use the same account as well as @colin-axner suggests